package.json

```diff
@@ -87,7 +87,7 @@
     "react-native-webview": "11.13.1",
     "react-redux": "7.2.5",
     "redux": "4.1.1",
-    "redux-saga": "1.1.3",
+    "redux-saga": "0.16.2",
```

Comment:
Any side effects to watch out for by reverting this verison?

Reply:
I needed to make a small change at the root saga, but we only recently added `redux-saga` to this repo. There is only one other saga, the keyboard open/close state. I tested that it's still working!